src/main/java/org/lesson43/dao/StudentDAO.java

```diff
@@ -0,0 +1,108 @@
+package org.lesson43.dao;
+
+import org.lesson43.models.Student;
+import org.lesson43.postgres.driver.PostgresDriverManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class StudentDAO {
+    private PostgresDriverManager postgresDriverManager;
+
+    @Autowired
+    public void setPostgresDriverManager(PostgresDriverManager postgresDriverManager) {
```

Comment:
Setters should be placed at the end of the class

Reply:
ok